src/main/resources/openapi/transcriptions.yaml

```diff
@@ -112,6 +112,66 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/TranscriptionRequestDetails'
+
+    patch:
+      tags:
+        - Transcription
+      summary: Update multiple transcriptions
+      operationId: updateTranscriptions
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/UpdateTranscriptionsItem'
+        '400':
+          description: Bad Request Error
+          content:
+            application/json+problem:
+              schema:
+                allOf:
+                  - $ref: './problem.yaml'
+                  - type: object
+                    required:
+                      - type
+                    properties:
+                      type:
+                        $ref: '#/components/schemas/UpdateTranscriptions400ErrorCode'
+              example:
```

Comment:
This example either needs to be updated to reflect the new error response, whatever that looks like now
or removed in my opinion

```
{
   "type":"TRANSCRIPTION_UT-PP-400",
   "title":"Failed to update some of the transcriptions",
   "status":400,
   "partial_failure":"{\"transcriptions\":[{\"transcription_id\":100,\"hide_request_from_requestor\":true}]}"
} 

```

Reply:
fixed